src/app/item-review/item-review.scss

```diff
@@ -91,5 +91,76 @@
 
   &--chart-container {
     width: 95%;
+    display: flex;
+  }
+
+  &--review-chart {
+    &.stats {
+      width: 100%;
+      display: table;
+      border-spacing: 0 3px;
```

Comment:
Did we need to copy/paste *all* the stats styles? Couldn't you just override the properties you care about?